src/System.Drawing.Primitives/src/System/Drawing/KnownColorTable.cs

```diff
@@ -21,11 +21,7 @@ private static void EnsureColorTable()
 
         private static void InitColorTable()
         {
-            int[] values = new int[(int)KnownColor.MenuHighlight + 1];
-
-            // system
-            //
-            UpdateSystemColors(values);
+            int[] values = new int[(int)KnownColor.LastColor + 1];
```

Comment:
Your right that is more general and I will update just to be sure we don't accidentally set it to anything beyond zero.